src/test/java/org/folio/edge/rtac/utils/RtacUtilTest.java

```diff
@@ -0,0 +1,81 @@
+package org.folio.edge.rtac.utils;
+
+import static io.vertx.core.http.HttpHeaders.ACCEPT;
+import static org.folio.edge.core.Constants.TEXT_PLAIN;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.vertx.core.MultiMap;
+import io.vertx.core.http.HttpServerRequest;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+
+@RunWith(VertxUnitRunner.class)
+public class RtacUtilTest {
+
+  @Mock
+  private static HttpServerRequest rtacRequest;
+  @Mock
+  private static MultiMap rtacHeaders;
+
+  private static List<String> supportedAcceptHeaders = Arrays
+      .asList(RtacMimeTypeEnum.getAllTypesAsString());
+  private static List<String> unsupportedAcceptHeaders = Arrays.asList(TEXT_PLAIN);
+
+  @BeforeClass
+  public static void beforeClassRtacUtils() {
+    // Creates mock of RTAC request nd headers
+    rtacRequest = mock(HttpServerRequest.class);
+    rtacHeaders = mock(MultiMap.class);
+
+    // Define mock object's behavior
+    when(rtacRequest.headers()).thenReturn(rtacHeaders);
+  }
+
+  @Test
+  public void shouldSupportedTypePassedWhenClientSpecifiedSupportedType() {
```

Comment:
Does this test check that when the request includes both XML and JSON then the request is considered supported?

Reply:
Yes, it tests all cases of supported headers which are specified in the RtacMimeTypeEnum. These are: 
- no headers case;
- */* all headers;
- application/xml or text/xml;
- application/json.
So by fact this test just verify that main headers support checking works well.

Reply:
Which of those scenarios does this specific test cover?

Reply:
It  tests most expected scenario when there any of acceptable headers were sent. I mean just testing that checking for supported headers was succeed.

Reply:
I don't understand what succeed means in this case.

Do you mean that the accept header provided by the client is acceptable to the RTAC module?

Reply:
As it turned out, not all the headers are acceptable by RTAC of course, I believe only some of defined as "supported" here can be processed by RTAC. For example json, "no headers at all" and wildcard all (*/*). Rest of them, all XML related will lead to error from RTAC side. As for this test in particular, I renamed it by the way, currently it has a name "shouldTreatRequestAsSupportedWhenClientSpecifiedSupportedType", it just tests scenario when any "supported" were passed in request, i.e. to be more precise - all the supported. But  by implelemntation,  checking considered as passed after first valid entry from those "supported list".

Reply:
Thank you for renaming the tests to make the intent clearer